docs/README.md

```diff
@@ -0,0 +1,42 @@
+# Dashboard Design Recommendations
+
+We recommend following some standards when building dashboards, to keep the visualization looking good and making them better for usability. 
+
+## Time Series Panels
+
+* For non duration metric panels (counter/gauge): Fully opaque charts:
```

Comment:
```suggestion
* For generic non-histogram metric panels (counter/gauge): Fully opaque area charts that give you visibility of the entire data even in darker backgrounds, and provide greater contrast when selecting result series as well.
```

Reply:
Hi thanks, added the recommendations just with minor changes, when you have the chance please review 🙏